.prettierrc

```diff
@@ -1,4 +1,7 @@
 {
+  "arrowParens": "always",
   "singleQuote": true,
-  "trailingComma": "all"
-}
+  "tabWidth": 2,
+  "printWidth": 80,
+  "trailingComma": "none"
```

Comment:
nit: why change this?

Reply:
I copied this from one of another projects, but actually can easily revert this change or use the right config that is most commonly used in our projects :shrug:

Reply:
Updated, used the https://github.com/grafana/k6-jslib-aws/blob/main/.prettierrc.json

Reply:
This does make quite a lot of whitespace changes and remove `;` at end and other strange things.

I wonder if we can shelf *that* particular change for now, just to merge this as is and then discuss this separately.

We do use https://github.com/grafana/k6-docs/blob/main/docs/sources/.prettierrc in the docs ... which I would expect is the one more people have seen and looked at.